.changeset/selfish-ducks-sell.md

```diff
@@ -0,0 +1,19 @@
+---
+'@evidence-dev/faker-datasource': minor
+'@evidence-dev/mssql': minor
+'@evidence-dev/snowflake': minor
+'@evidence-dev/evidence': minor
+'@evidence-dev/component-utilities': minor
+'@evidence-dev/db-orchestrator': minor
+'@evidence-dev/plugin-connector': minor
+'@evidence-dev/preprocess': minor
+'@evidence-dev/query-store': minor
+'@evidence-dev/sdk': minor
+'@evidence-dev/universal-sql': minor
+'@evidence-dev/core-components': minor
+'my-evidence-project': minor
+'@evidence-dev/components': minor
+'evidence-test-environment': minor
```

Comment:
This should only be a `patch` change for the impacted package (in this case `preprocess`)

Reply:
@ItsMeBrianD 
sure will change it, shall I also revert the pnpm.lock file commit.
Without commiting pnpm.lock file, previews were not getting deployed.

Reply:
No the lock file is okay to include!

Reply:
Done